testing/kokoro/decrypt-secrets.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
This name doesn't really make sense. It's not "decrypting" the secrets.

Reply:
The name was borrowed from Python's repo, but I'm open to new names. I imagine they chose it because the data sits encrypted in Secrets Manager :)